packages/ui-components/src/chat/message.tsx

```diff
@@ -117,6 +118,21 @@ export interface MessageProps extends InteractiveComponentProps<Message> {
     onDelete?: (id: string) => void | Promise<void>;
 }
 
+/**
+ * A function to get the formatted timestamp to display in the submitted message
+ */
+export function getFormattedTimestamp(): string {
```

Comment:
By the way I think we already have date-fns installed so you don't need to manually do the formatting